web/forms/binding_a_boolean_field_to_a_checkbox/my_element.html

```diff
@@ -1,11 +1,11 @@
-<polymer-element name="my-element">
- <template>
+<dom-module id="my-element">
+  <template>
     <div>
       <label>
-        <template if="{{checked}}">Uncheck</template>
-        <template if="{{!checked}}">Check</template>
+        <template is="dom-if" if="{{checked}}">Uncheck</template>
```

Comment:
Not necessary to change in this example I don't think, but this is not actually a good use of `dom-if`. A computed binding would be a better choice here, something like:

```html
<label>{{checkedText(checked)}}</label><br>
```

```dart
   @eventHandler
   String checkedText(bool isChecked) => isChecked ? 'Uncheck' : 'Check';
```